### ISSUE TITLE
At Program Start the CLI greets the user

### ISSUE THREAD

**Issue by lukaswolfrum** [amosproj/amos2022ss07-nft-playbook#9]
## User story
1. As a power user
3. I want / need to be able to see that the program is ready for my input
4. So that I am able to start working with it 

## Acceptance criteria
* program greets the user when it starts
* program shows basic information about its usage
* waits for further input

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

actual size = 1
